_pages/volunteer.md

```diff
@@ -0,0 +1,97 @@
+
+## pyOpenSci volunteers build skills and community  
+
+When you volunteer with pyOpenSci, you’re not only giving back to the community, but also:  
+
+- **Learning new skills:** you don’t have to be a Python expert to get involved with pyOpenSci. We can help you level up your packaging game and learn how to constructively review both code and copy through contributions to our online learning resources.   
+- **Connecting with peers:** our [public Discourse forum](https://pyopensci.discourse.group/) is a great place to ask and answer questions, as well as discuss packaging and open science with other Pythonistas.  
+- **Being recognized** on the [pyOpenSci website](https://www.pyopensci.org/our-community/index.html) and [GitHub repositories](https://github.com/pyOpenSci): your contribution matters, and we want to ensure your work is recognized and celebrated in a public forum.  
+
+## Volunteer opportunities with pyOpenSci
+
+There are many ways to get involved with pyOpenSci! We’re always looking for folks to:  
+
+{% include feature_row_pyos id="help-us" %}
+
+## Share your expertise and experience with the broader community through blogging  
+
+And last but not least, we’d also love for you to be a guest blogger on the [pyOpenSci blog](https://www.pyopensci.org/blog/index.html)! If you’d like to write about a pyOpenSci package, your experiences with pyOpenSci, or how you’re using free and open Python tools in your scientific endeavors, we’d love to hear from you! Email our Community team at [media@pyopensci.org](media@pyopensci.org) for more information.   
+
+## Learn more about volunteering with pyOpenSci  
+
+If you still have questions about volunteer roles at pyOpenSci, email our Community team at [media@pyopensci.org](media@pyopensci.org), and we’d be more than happy to help! You can also ask questions and network with the broader pyOpenSci community on our [Discourse forum](https://pyopensci.discourse.group/).  
+
+## Let's connect!
+
+- [<i class="fa-brands fa-discourse"></i> Discourse](https://pyopensci.discourse.group/)
+- [<i class="fa-brands fa-mastodon"></i> Mastodon](https://fosstodon.org/@pyopensci)
+- [<i class="fa-solid fa-cloud"></i> Bluesky](https://bsky.app/profile/pyopensci.bsky.social)
+- [<i class="fa-brands fa-linkedin"></i> LinkedIn](https://www.linkedin.com/company/pyopensci)
+- [<i class="fa-brands fa-github"></i> GitHub](https://github.com/pyOpenSci)
+
+## Meet the PyOpenSci community contributors
```

Comment:
@kierisi i love this idea but it feels like a lot to have the full isotope filter here. what do you think about adding something like we have on the home page . i thijnk that will look nicer and be lighter weight to load. we can then add a button that says "view all contributors". i'll go ahead and add this now and if we don't like it we can always remove it. 
<img width="1321" alt="Screenshot 2024-01-03 at 10 27 19 AM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/0d3f448e-f056-4bc6-a8da-569df388cd21">

Reply:
works for me!